authserver/oauth2/rfc6749/authorization_server.py

```diff
@@ -8,6 +8,50 @@
 from authlib.common.encoding import to_unicode
 from authlib.oauth2.rfc6749 import InvalidGrantError, OAuth2Error
 
+import jwt
+
+from authserver.config import ConfigurationFactory
+from datetime import datetime, timedelta
+class BrighthiveJWT(object):
+    def __init__(self):
+        self.private_key = ConfigurationFactory.from_env().signature_key
+
+    def generated_claims(self) -> object:
+        return {
+            "iss": "brighthive-authserver",
+            "aud": [
+                "brighthive-data-trust-manager",
+                "brighthive-data-catalog-manager",
+                "brighthive-governance-api",
+                "brighthive-permissions-service",
+                "brighthive-data-uploader-service"
+                "brighthive-authserver"
+            ],
+            "iat": datetime.utcnow(),
+            "exp": datetime.utcnow() + timedelta(24) # TODO: Set to now + 15 minutes
+        }
+
+    def make_jwt(self, json_claims: object) -> object:
+        claims = self.generated_claims()
+        claims.update(json_claims)
+
+        jwt_token = jwt.encode(claims, self.private_key, algorithm='RS256')
+
+        print(f'data {claims}')
+        print(f'jwt_token {jwt_token}')
+
+        return jwt_token
+
+
+def generate_jwt(access_token):
+    try:
+        json_claims = {"brighthive-access-token": access_token, "brighthive-org-role": "???"}
+        a_jwt = BrighthiveJWT().make_jwt(json_claims)
+    except Exception as e:
+        print('JWT generation exception:', e)
+        return 'none'
+
+    return a_jwt
```

Comment:
What is with the "???"?

Reply:
just a placeholder for the lambda service